### DISCUSSION TITLE
Drawimage/Imagecomposesnapshot/vertexdisplacementmap conflict

### DISCUSSION THREAD

**Discussion by jaalibandar** [cables-gl/cables_docs#320] [! Welcome - readme first!]
Link to public patch here: https://cables.gl/edit/BE-ELa 

Problem Description:
DrawImage doesn't work as it is supposed to. 
See left most viztexture in the patch above. Refer to the 'bug area'.
You need to right click on the canvas to trigger the TriggerOnce op for changes to take effect.

Findings: 
1. Disconnect Triggeronce and connect imagecompose directly to sequence to make it work 
2. Remove VertexDisplacementMap and it works 
3. Instead of using drawimage, connect the texture directly to imagecompose

**Comment by pandrr** [comment]
you are using images in vertexdisplacement that are not rendered yet, look at the order of outputs of the sequence

**Comment by pandrr** [comment]
![Screenshot 2022-08-18 at 13 46 00](https://user-images.githubusercontent.com/98792/185387082-b5fbadfa-521b-42ca-9c21-751fad4056df.png)

this way it works execution order is from left to right

**Reply by jaalibandar** [reply]
thank you
